pkg/client/client.go

```diff
@@ -26,13 +26,14 @@ type Client struct {
 
 // Config is the configuration definition for connecting to a Falco gRPC server.
 type Config struct {
-	Hostname       string
-	Port           uint16
-	CertFile       string
-	KeyFile        string
-	CARootFile     string
-	UnixSocketPath string
-	DialOptions    []grpc.DialOption
+	Hostname       					string
+	Port           					uint16
+	CertFile       					string
+	KeyFile        					string
+	CARootFile     					string
+	UnixSocketPath 					string
+	DialOptions    					[]grpc.DialOption
+	InsecureSkipMutualTLSAuth       bool
```

Comment:
While I understand the desire to have all the types aligned, it seems there's an issue here.

```suggestion
	Hostname 			string
	Port 				uint16
	CertFile 			string
	KeyFile 			string
	CARootFile 			string
	UnixSocketPath 			string
	DialOptions 			[]grpc.DialOption
	InsecureSkipMutualTLSAuth 	bool
```

Reply:
Please note that running `go fmt` will modify that part to use spaces only for the alignment.